src/components/footer/footer.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../scss/helsinki/custom-styles';
+@import '../../scss/helsinki/variables';
+
+.koroFooter{
+      width: 100%;
+      height: $line-height-computed *11;
```

Comment:
I don't think the footers height should depend on variables like this, just write 11rem.